Prox/Prox/RemoteConfigKeys.swift

```diff
@@ -87,16 +87,34 @@ class RemoteConfigProperty<T> {
 
     var value: T {
         let remoteConfig = FIRRemoteConfig.remoteConfig()
+
+        // rcv is never nil even if there is no key.
+        // rcv.numberValue? never fails either (in light of an invalid number, it defaults to zero).
+        // So, we get the string value, and try parsing it ourselves.
         let rcv = remoteConfig[key]
-        if let value = convert(rcv) {
-            return value
+        if let string = rcv.stringValue {
+            if let value = convert(string) {
+                return value
+            } else if string != "" {
+                // i.e. no default exists, and no value is specified.
+                NSLog("RemoteConfigKeys: Existing value for \(key) is not a valid \(type(of: defaultValue))")
+            }
         }
-        if let rcv = remoteConfig.defaultValue(forKey: key, namespace: nil),
-            let value = convert(rcv) {
-            print("RemoteConfigKeys: Default value for \(key) from embedded plist")
+
+        // TODO: save the last successful value in UserDefaults.standard, then fall back to it here,
```

Comment:
yes, me too.

Comment added:

        // This could lead to all sorts of caching related bugs,
        // but would ameliorate the jarring application of a way out of date default.